content/influxdb/v2.0/api-guide/client-libraries/nodejs/install.md

```diff
@@ -50,19 +65,29 @@ Add both as dependencies of your project.
    npm install --save @influxdata/influxdb-client-apis
    ```
 
+## Next steps
+
+Once you've installed the Javascript client library, you're ready to [write data](/influxdb/v2.0/api-guide/client-libraries/nodejs/write.md) to InfluxDB or [get started](#get-started-with-examples) with other examples from the client library.
+
 ## Get started with examples 
 
 {{% note %}}
 The client examples include an [`env`](https://github.com/influxdata/influxdb-client-js/blob/master/examples/env.js) module for accessing your InfluxDB properties from environment variables or from `env.js`.
-The examples will use these to interact with the InfluxDB API.
+The examples use these properties to interact with the InfluxDB API.
 {{% /note %}}
 
 1. Set environment variables or update `env.js` with your InfluxDB [bucket](/influxdb/v2.0/organizations/buckets/), [organization](/influxdb/v2.0/organizations/), [token](/influxdb/v2.0/security/tokens/), and [url](/influxdb/v2.0/urls).
 
    ```sh
-   # Environment variables
-   {{< api/v2dot0/env >}}
+   export INFLUX_URL=http://localhost:8086
+   export INFLUX_TOKEN=YOUR_API_TOKEN
+   export INFLUX_ORG=YOUR_ORG
+   export INFLUX_BUCKET=YOUR_BUCKET
    ```
+   Replace the following:
+   - *`YOUR_API_TOKEN`*: InfluxDB API token.
+   - *`YOUR_ORG`*: InfluxDB organization ID.
+   - *`YOUR_BUCKET`*: InfluxDB bucket name.
```

Comment:
If the descriptions aren't complete sentences, I don't include periods.
```suggestion
   - *`YOUR_API_TOKEN`*: InfluxDB API token
   - *`YOUR_ORG`*: InfluxDB organization ID
   - *`YOUR_BUCKET`*: InfluxDB bucket name
```

Reply:
Thanks. I modeled what I had in the previous PR after [Google's pattern](https://developers.google.com/style/placeholders?hl=en#two-or-more-placeholders), which is more wordy. Are we intentionally breaking that rule?